models.yml

```diff
@@ -14,6 +14,409 @@ basel:
   supportedParameters:
     - contactReduction
     - r0
+  supportedRegions:
+    - AF
+    - AL
+    - DZ
+    - AD
+    - AG
+    - AR
+    - AM
+    - AU
+    - AT
+    - AZ
+    - BS
+    - BH
+    - BD
+    - BB
+    - BY
+    - BE
+    - BZ
+    - BJ
+    - BT
+    - BO
+    - BA
+    - BW
+    - BR
+    - BN
+    - BG
+    - BF
+    - BI
+    - CV
+    - KH
+    - CM
+    - CA:
+        - 'CA-AB'
+        - 'CA-BC'
+        - 'CA-MB'
+        - 'CA-NT'
+        - 'CA-NB'
+        - 'CA-NL'
+        - 'CA-NS'
+        - 'CA-NU'
+        - 'CA-ON'
+        - 'CA-PE'
+        - 'CA-QC'
+        - 'CA-SK'
+        - 'CA-YT'
+    - CF
+    - CL
+    - CN
+    - CO
+    - KM
+    - CR
+    - HR
+    - CU
+    - CY
+    - CZ
+    - DK
+    - DJ
+    - DM
+    - DO
+    - EC
+    - EG
+    - SV
+    - GQ
+    - EE
+    - SZ
+    - ET
+    - FO
+    - FJ
+    - FI
+    - FR:
```

Comment:
I think we may also need to add an “All” region or a “_self” region for countries that support sub regions. The reason is that for imperial, we support state’s but not the US as a whole. So we need to distinguish that in the list.